pandas/tests/test_frame.py

```diff
@@ -5545,6 +5545,8 @@ def test_itertuples(self):
         dfaa = df[['a', 'a']]
         self.assertEqual(list(dfaa.itertuples()), [(0, 1, 1), (1, 2, 2), (2, 3, 3)])
 
+        self.assertEqual(type(next(df.itertuples(name=None))), tuple)
```

Comment:
we don't use ``type(...)`` is not idiomatic. use ``isinstance`` and see why what you are doing is wrong.